frontend/src/errorHandling/RenderErrorHandler/RenderErrorHandler.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC, useContext } from 'react';
+import Button from 'antd-button-color';
+import { Result } from 'antd';
+import { ErrorContext } from '../ErrorContext';
+import { ErrorItem } from '../ErrorHandler';
+import { CustomError } from '../utils';
+
+export interface IRenderErrorHandlerProps {
+  className?: string;
+  errors: CustomError[];
+}
+
+const RenderErrorHandler: FC<IRenderErrorHandlerProps> = ({ ...props }) => {
+  const { className, errors } = props;
+  const { flushRenderError } = useContext(ErrorContext);
+  const errorsMapped = errors.map((e, i) => <ErrorItem key={i} item={e} />);
+  return (
+    <div
+      className={`flex flex-col h-full w-full justify-center items-center ${className}`}
+    >
+      <div className="flex w-full justify-center">
+        <Result
+          className="p-0"
+          //It's not a real 500 error, that's just for visualize a cute image with the component
+          status="500"
+          title="Application Error."
+          //subTitle={'Sorry, something went wrong.'}
+        />
```

Comment:
Hasn't this been done by @cheina97 already?

Reply:
i renamed the component and changed some content, and after latest push i removed the old francesco one